### PR TITLE
Add some test data for PTS and a quick test checking PTS to PPTS links

### PR DESCRIPTION
**Pull request by ilakyapal**

### CODE REVIEW

File: schemaless/create_uuid_map.py [hunk 1]
```diff
@@ -53,6 +55,10 @@ def from_files(cls, schemaless_file, uuid_map_file):
                         parents.extend(
                             record['planning_case_number'].split(","))
 
+                if source == MOHCD.NAME:
```
Comment (sbuss) [inline schemaless/create_uuid_map.py]: duplicated (see above)

Comment (ilakyapal) [reply schemaless/create_uuid_map.py]: Ah good catch! thanks :)

**Comment by ilakyapal** [comment]
Fixes https://github.com/sfcpc/housing-dashboard/issues/37

**Comment by ilakyapal** [comment]
Fixes https://github.com/sfcpc/housing-dashboard/issues/34

**Pull request merged**